src/spec/monasca_client_spec.js

```diff
@@ -0,0 +1,78 @@
+import MonascaClient from "../components/monasca_client.js"
+
+// listDimensionNames() {
+//     return this._get('/v2.0/metrics/dimensions/names')
+//       .then(resp => resp.data.elements.map(e => e.dimension_name))
+//       .catch(err => { throw err; });
+// }
+
+
+function backendSrvMock(){
+
+}
+
+function datasourceSrvMock(){
+
+}
```

Comment:
Will backendSrvMock and datasourceSrvMock be necessary after removing them as inputs?